src/adapters/bidderFactory.js

```diff
@@ -8,22 +8,17 @@ import {isValidVideoBid} from '../video.js';
 import CONSTANTS from '../constants.json';
 import * as events from '../events.js';
 import {includes} from '../polyfill.js';
-import {ajax} from '../ajax.js';
 import {
-  deepAccess,
   delayExecution,
-  flatten,
   isArray,
   isPlainObject,
   logError,
   logWarn,
   parseQueryStringParameters,
   parseSizesInput,
-  timestamp,
   uniques
 } from '../utils.js';
-import {ADPOD} from '../mediaTypes.js';
-import {getHook, hook} from '../hook.js';
+import {hook} from '../hook.js';
```

Comment:
`storage` (just below the fold) should be unnecessary now, I'd like to get rid of it since it was one of the few grey area cases where we have "lax" GDPR enforcement.

I think it's kept alive by tests, otherwise the linter should have complained.

Reply:
I see what you mean.  I'll make another push to remove that and check for any tests.

Reply:
Both changes have been pushed.  I searched through the project and I couldn't find any imported references or uses of the `storage` export from the bidderFactory file.  If you think there some references still, please let me know.